docs/_reporting/hubs/configure-scopes.md

```diff
@@ -36,43 +36,6 @@ For the most seamless experience, we recommend allowing FinOps hubs to manage ex
 
 <br>
 
-
-## 🛠️ Configure exports manually
```

Comment:
Did you mean to get rid of this? Don't we need it?

Reply:
yes, I was getting a merge conflict